insets/src/main/java/dev/chrisbanes/accompanist/insets/Insets.kt

```diff
@@ -105,22 +105,74 @@ val AmbientWindowInsets = staticAmbientOf<WindowInsets> {
 }
 
 /**
- * Applies any [WindowInsetsCompat] values to [AmbientWindowInsets], which are then available
- * within [content].
+ * This class sets up the necessary listeners on the given [view] to be able to observe
+ * [WindowInsetsCompat] instances dispatched by the system.
  *
- * @param consumeWindowInsets Whether to consume any [WindowInsetsCompat]s which are dispatched to
- * the host view. Defaults to `true`.
+ * This class is useful for when you prefer to handle the ownership of the [WindowInsets]
+ * yourself. One example of this is if you find yourself using [ProvideWindowInsets] in fragments.
+ *
+ * It is convenient to use [ProvideWindowInsets] in fragments, but that can result in a
+ * delay in the initial inset update, which results in a visual flicker.
+ * See [this issue](https://github.com/chrisbanes/accompanist/issues/155) for more information.
+ *
+ * The alternative is for fragments to manage the [WindowInsets] themselves, like so:
+ *
+ * ```
+ * override fun onCreateView(
+ *     inflater: LayoutInflater,
+ *     container: ViewGroup?,
+ *     savedInstanceState: Bundle?
+ * ): View = ComposeView(requireContext()).apply {
+ *     layoutParams = LayoutParams(MATCH_PARENT, MATCH_PARENT)
+ *
+ *     // Create an ViewWindowInsetObserver using this view
+ *     val observer = ViewWindowInsetObserver(this)
+ *
+ *     // Call start() to start listening now.
+ *     // The WindowInsets instance is returned to us.
+ *     val windowInsets = observer.observe()
```

Comment:
This should be `start` right?

```suggestion
 *     val windowInsets = observer.start()
```

Reply:
Yep, woops!